client/src/templates/Challenges/generic/show.tsx

```diff
@@ -210,6 +221,20 @@ const ShowGeneric = ({
 
   const sceneSubject = new SceneSubject();
 
+  // interactive editor
+  const hasInteractiveEditor = nodules?.some(
+    nodule => nodule.type === 'interactiveEditor'
+  );
+
+  const [showInteractiveEditor, setShowInteractiveEditor] = useState(
+    () => (store.get('showInteractiveEditor') as boolean | null) ?? false
```

Comment:
```suggestion
    () => !!store.get('showInteractiveEditor')
```
Nit: same semantics, but a little neater imo.